Pythonwin/pywin/framework/scriptutils.py

```diff
@@ -497,7 +499,7 @@ def CheckFile():
     without actually executing any code (ie, by compiling only)
     """
     try:
-        pathName = GetActiveFileName()
+        pathName = str(GetActiveFileName())
```

Comment:
This does change the error at line 510 from:
```
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
TypeError: expected str, bytes or os.PathLike object, not NoneType
```
to the following print:
`Can't open file 'None' - [Errno 2] No such file or directory: 'None'`